.github/workflows/zips.yml

```diff
@@ -208,3 +214,27 @@ jobs:
           comment-id: ${{ steps.find-comment.outputs.comment-id }}
           edit-mode: replace
           body: Build files for ${{ github.event.pull_request.head.sha }} have been deleted.
+
+  publish-to-wporg:
+    name: Publish to WordPress.org
+    runs-on: ubuntu-latest
+    environment: wordpress-plugin-svn
+    if: github.event_name == 'workflow_dispatch' && ( github.ref_type == 'tag' || inputs.release_version != '' )
+    needs: build-zips
+    timeout-minutes: 20
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: zip-files
+          path: ${{ github.ref }}
+      - name: Extract
+        run: |
+          unzip google-site-kit.zip
+          mv google-site-kit/* .
+          rm -rf google-site-kit *.zip
+          echo .distignore > .distignore
+      - uses: 10up/action-wordpress-plugin-deploy@stable
+        env:
+          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          VERSION: ${{ inputs.release_version }}
```

Comment:
What if `github.ref_type == 'tag'` but there was no `release_version` provided? Can we default the `release_version` to the tag? The `if` above requires `github.ref_type == 'tag' || inputs.release_version != ''`, so if we don't use the `tag` then it should probably also require the version input.

Reply:
The documentation says that if we don't use version, then it will default to tag name by default:

```
VERSION - Defaults to the tag name. We do not recommend setting this except for testing purposes.
```

Wondering why they don't recommend setting it 🤔.

Reply:
Yeah, I'm not sure that would work for us because we're triggering it with `workflow_dispatch` rather than the `push` for tags 🤔 I like the idea of using the tag as the version to avoid user error. WDYT?

https://github.com/10up/action-wordpress-plugin-deploy/blob/977dc0cce7451fdd3ce4830920f75118dfc57002/deploy.sh#L33-L37

Reply:
When we manually trigger it with `workflow_dispatch`, we can select which branch or tag we want to use. So, if we have a release tag already created, we can always select it and leave the version field empty.

I think we should keep the version input too, to have an ability to force a specific version to be published.